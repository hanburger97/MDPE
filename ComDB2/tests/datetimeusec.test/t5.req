set timezone UTC
select cast(dt as TEXT), cast(ds as TEXT), id from dtus order by id
