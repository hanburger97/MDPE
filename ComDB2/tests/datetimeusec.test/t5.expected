(cast(dt as TEXT)='1970-01-01T010000.246000 UTC', cast(ds as TEXT)=NULL, id=1)
(cast(dt as TEXT)='1970-01-01T010000.246000 UTC', cast(ds as TEXT)=NULL, id=2)
(cast(dt as TEXT)='1970-01-01T020000.246000 UTC', cast(ds as TEXT)=NULL, id=3)
(cast(dt as TEXT)='2015-02-23T080000.246000 UTC', cast(ds as TEXT)=NULL, id=4)
(cast(dt as TEXT)='2015-02-23T080000.246000 UTC', cast(ds as TEXT)=NULL, id=5)
(cast(dt as TEXT)='2015-02-23T080000.246000 UTC', cast(ds as TEXT)=NULL, id=6)
(cast(dt as TEXT)=NULL, cast(ds as TEXT)='0 01:00:00.246000', id=7)
(cast(dt as TEXT)=NULL, cast(ds as TEXT)='0 01:00:00.246000', id=8)
(cast(dt as TEXT)=NULL, cast(ds as TEXT)='0 02:00:00.246000', id=9)
(cast(dt as TEXT)=NULL, cast(ds as TEXT)='0 00:00:01.246000', id=10)
(cast(dt as TEXT)=NULL, cast(ds as TEXT)='0 00:00:01.246000', id=11)
(cast(dt as TEXT)=NULL, cast(ds as TEXT)='0 00:00:10.000000', id=12)
(cast(dt as TEXT)=NULL, cast(ds as TEXT)='10 00:00:00.123000', id=13)
(cast(dt as TEXT)=NULL, cast(ds as TEXT)='0 02:04:06.246000', id=14)
